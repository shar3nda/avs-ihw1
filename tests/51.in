3 10 5 -10 8 -8 9 3 0 -10