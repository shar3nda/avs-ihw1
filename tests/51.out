-10 8 -8 9 3 0 -10